### PR TITLE
[Monorepo] fix unit tests

### PR DESCRIPTION
**Pull request by dratwas**
## Description
This is a part of migration gutenberg-mobile to gutenberg repo.

In this PR I fixed native unit tests. 
I removed unused jest.config files and use configs from `test/native` directory

## How has this been tested?
`npm run native test` should run all native tests and all of them should pass.

## Types of changes
fix native tests after migration

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -212,6 +212,3 @@ jobs:
         local_dir: playground/dist
         on:
           branch: master
-
```
Comment (gziolo) [inline .travis.yml]: I think this landed with the related work done by @hypest.

File: test/native/jest.config.js [hunk 2]
```diff
@@ -31,11 +31,12 @@ module.exports = {
 	testEnvironment: 'jsdom',
 	testMatch: [
 		'**/test/*.native.[jt]s?(x)',
+		'<rootDir>/packages/react-native-editor/src/**/__tests__/**/*.[jt]s?(x)',
```
Comment (gziolo) [inline test/native/jest.config.js]: I left a similar comment in another file, this is fine as a temp solution, but in the long run we should adjust test names to make it seamless for both the web and mobile. I mean, it should be based on conventions so we don't have to tweak config files too much.

File: test/native/setup.js [hunk 3]
```diff
@@ -3,7 +3,7 @@
  */
 import { NativeModules } from 'react-native';
 
-jest.mock( '@wordspress/react-native-bridge', () => {
```
Comment (gziolo) [inline test/native/setup.js]: 😅

File: .travis.yml (hunk 1 above)
Comment (dratwas) [reply .travis.yml]: Yeah, will have to resolve conflicts later

**Comment by Tug** [comment]
Closing this one as unit tests are green since https://github.com/WordPress/gutenberg/pull/18531 was merged and we will keep them this way for all future work that targets the feature branch.

**Pull request closed without merging**